testutils/build.gradle

```diff
@@ -44,6 +53,37 @@ android {
 
 }
 
+configurations {
+    repackage // configuration to repackage a jar
+}
+
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+
+/* a task to create the relocated libs */
+task repackageJar(type: ShadowJar) {
+    def originalPkg = 'org.apache.http' // lib to relocate
+    def shadowPkg = 'cz.msebera.android.httpclient' // the lib that will be substituted
+    relocate originalPkg, shadowPkg
+
+    // excluding these from package to allow repackaging
+    exclude 'META-INF/*.MF'
+    exclude 'META-INF/*.RSA'
+    exclude 'META-INF/*.SF'
+
+    dependencies {
+        // we are already pulling these from other dependencies
+        exclude(dependency("com.google.guava:guava:.*"))
+        exclude(dependency("org.ow2.asm:asm:.*"))
```

Comment:
OSS component reviewed?

Reply:
Nope, Thanks for pointing out. Will do that. For test only dependencies, that actually just means registering the component in Component Governance.

Reply:
This was is actually just an exclude though -  not a dep being added. So this shouldn't need a review